_episodes_rmd/12-time-series-raster.Rmd

```diff
@@ -235,23 +247,28 @@ quickly apply this factor using raster math on the entire stack as follows:
 
 `raster_stack_object_name / 10000`
 
-> ## Data Tip
-> We can make this plot
-> even prettier by fixing the individual tile names, adding an plot title and by
-> using the (`levelplot`) function. This is covered in the NEON Data Skills
-> [Plot Time Series Rasters in R ]({{ site.baseurl }}/13-plot-time-series-rasters-in-r/)
-> episode.
-{: .callout}
 
 ```{r apply-scale-factor }
 
 # apply scale factor to data
 NDVI_HARV_stack <- NDVI_HARV_stack/10000
 # plot stack with scale factor applied
-# apply scale factor to limits to ensure uniform plottin
-plot(NDVI_HARV_stack,
-     zlim = c(.15, 1),
-     nc = 4)
+# convert to a df for plotting in two steps,
```

Comment:
These lines look identical to lines 217-225. Can they be removed or is there a difference I'm not seeing?

Reply:
Yes it is a bit heavy but because we changed NDVI_HARV_stack (divided by 10000), we need to recreate NDVI_HARV_stack _df again, isn't it?